### ISSUE TITLE
logo.svg missing?

### ISSUE THREAD

**Issue by regio760** [wet-boew/theme-gc-intranet#156]
In the intranet theme, the splashpage code includes a reference to a log.svg file that seems to not be in the assets folder that it refers to - is that missing or is that deliberate?

**Comment by LaurentGoderre** [comment]
The file is not missing, it's the actual page that is incorrect and should not be used.

**Comment by RobJohnston** [comment]
Can I get a citation on that?  What if it's an intranet application... don't they have splash pages?

**Comment by regio760** [comment]
sorry, the file I meant was logo.svg.  apologies for the typo

**Comment by LaurentGoderre** [comment]
@RobJohnston no. The demos have a sample splash page to be used. The file that reference the logo.svg is a residual file carried over from the core repo

**Comment by regio760** [comment]
The file I am referring to is splashpage-en.html from the themes-dist-4.0.20-theme-gc-intranet folder.  It contains the logo.svg.  Is there a different splashpage I should be using?

**Comment by LaurentGoderre** [comment]
Hmmm. I guess the splash page was never implemented for this theme

**Comment by regio760** [comment]
The file I'm looking at came with the download of the theme for 4.0.20.  When you say it wasn't implemented for this theme, does that mean what I found is not valid?

**Comment by LaurentGoderre** [comment]
It is using the splash markup from the WET theme so it's not specific to this theme no

**Comment by regio760** [comment]
Do you know if that is normal/intended or if this was an oversite that might be updated at some point?  This is the theme I have to use in the next upgrade to our project so I need to be clear on the details.

**Comment by LaurentGoderre** [comment]
It probably should be update to somewhat resemble the GCWU theme

**Comment by regio760** [comment]
Is there something I should be doing to request this splash page?

**Comment by LaurentGoderre** [comment]
I don't know who maintains this theme now /cc @plansmash
